packages/@aws-cdk/cicd/README.md

```diff
@@ -0,0 +1,31 @@
+## Continuous Integration / Continuous Delivery for CDK Applications
+
+### Getting Started
+In order to add the `PipelineDeployStackAction` to your *CodePipeline*, you need to have a *CodePipeline* artifact that
```

Comment:
Can you provide a few words of overview? Something like "This library includes a CodePipeline action for deploying AWS CDK applications".

Also, can you provide a full example that contains a build step and how to wire the output from the build to this action.

I also wonder how this plays out with how an app is organized? Usually one adds stacks to an `App` object. Where is the pipeline expected to be mastered and how are stacks going to passed into the pipeline? A minimal but complete example would be useful.

Reply:
Expanded on the example - I may want to look into making it a `.lit.ts` example, but that'll have to be later.